src/CILogViewer.php

```diff
@@ -78,39 +63,30 @@ public function __construct() {
         $this->init();
     }
 
-
     /**
      * Bootstrap the library
      * sets the configuration variables
      * @throws \Exception
      */
     private function init() {
+        $viewerConfig = config('CILogViewer');
 
-        if(!function_exists("get_instance")) {
-            throw new \Exception("This library works in a Code Igniter Project/Environment");
+        if($viewerConfig) {
+            if(isset($viewerConfig->viewPath)) {
+                $this->viewPath = $viewerConfig->viewPath;
+            }
+            if(isset($viewerConfig->logFilePattern)) {
+                $this->logFilePattern = $viewerConfig->logFilePattern;
+            }
```

Comment:
We're not taking note of the `$logFolderPath` here, which will allow users to specify different folder path.

Reply:
The idea is making use of the CI4  `App/Config/Logger.php` file which allows to specify a general purpose logger path (by default. `WRITEPATH . 'logs/'` ). If I understood correctly the purpose of logFolderPath this would be redundant, why duplicating this setting if CI4 already provides a variable for it ?

Reply:
I see the reasoning now, thanks.

Reply:
Ok **forget my last comment**. I was wrong as I assumed that the `path` would be accesible from the `Logger` class but that was a member of the `$handles` array that is passed as $config on the Logger constructor. So in summary is that `path` is not exposed.
But if you know how to get this value, them I think my previous comment would make total sense.
For now I'll do what you suggested.